app/action-links/server/lib/actionLinks.js

```diff
@@ -1,6 +1,16 @@
 import { Meteor } from 'meteor/meteor';
 
-import { Messages, Subscriptions } from '../../../models/server';
+import { getMessageForUser } from '../../../../server/lib/messages/getMessageForUser';
+
+function getMessage(messageId) {
```

Comment:
I think it'd be better to change this function name, because `getMessage` already exists in the `actionLinks` namespace